webapp/src/components/properties/link/link.scss

```diff
@@ -1,6 +1,8 @@
 .focalboard-body .property-link {
     &.url {
         padding-right: 2rem;
+        width: inherit;
```

Comment:
By the way why `width: inherit;` is needed here?

Reply:
Shouldn't be.